examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+"""
```

Comment:
another enhancement regarding local sphinx-build would be to allow single example as target (since some of the example execution times are long)

To build only one file I locally updated `conf.py` by adding `filename_pattern`

```
sphinx_gallery_conf = {
     # path to your examples scripts
     'examples_dirs': '../../examples',
     **'filename_pattern': '/plot_callosal_tract_profile.py',**
     # path where to save gallery generated examples
     'gallery_dirs': 'auto_examples',
     'image_scrapers': image_scrapers,
}
```